pkg/document/crdt/text.go

```diff
@@ -250,28 +250,58 @@ func (t *Text) Edit(
 func (t *Text) Style(
 	from,
 	to *RGATreeSplitNodePos,
+	latestCreatedAtMapByActor map[string]*time.Ticket,
 	attributes map[string]string,
 	executedAt *time.Ticket,
-) error {
+) (map[string]*time.Ticket, error) {
 	// 01. Split nodes with from and to
 	_, toRight, err := t.rgaTreeSplit.findNodeWithSplit(to, executedAt)
 	if err != nil {
-		return err
+		return nil, err
 	}
 	_, fromRight, err := t.rgaTreeSplit.findNodeWithSplit(from, executedAt)
 	if err != nil {
-		return err
+		return nil, err
 	}
 
 	// 02. style nodes between from and to
 	nodes := t.rgaTreeSplit.findBetween(fromRight, toRight)
+	createdAtMapByActor := make(map[string]*time.Ticket)
+	var toBeStyled []*RGATreeSplitNode[*TextValue]
+
 	for _, node := range nodes {
+		actorIDHex := node.id.createdAt.ActorIDHex()
+
+		var latestCreatedAt *time.Ticket
+		if latestCreatedAtMapByActor == nil {
```

Comment:
Style operations generated in the previous protocol version were setting `createdAtMap` as an empty map, which led to an issue where the `latestCreateAt` was set to `InitialTicket`, causing the style not to be applied.  To resolve this issue, we can set the `latestCreateAt` to `MaxTicket` when the map is empty. 

```suggestion
		if len(latestCreatedAtMapByActor) == 0 {
```